tedium.js

```diff
@@ -203,13 +290,14 @@ Promise.resolve().then(() => {
   const promises = [];
 
   for (const ghRepo of ghRepos) {
-    promises.push(rateLimit(100).then(() => {
+    promises.push(Promise.resolve().then(() => {
```

Comment:
This could potentially be simplified to `Promise.resolve((() => { ... })());`.

Even better, if `promiseAllWithProgress` accepts non-promises in the list (as `Promise.all` does), you can skip the `Promise.resolve`.